dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -24,17 +25,31 @@ def login
     elsif params[:platform_id]
       query_params = {platform_id: params[:platform_id]}
     else
-      return unauthorized_status
+      return log_unauthorized(
+        "Missing required parameters for LTI authentication",
+        {
+          client_id: params[:client_id],
+          issuer: params[:iss],
+          platform_id: params[:platform_id],
+        }
+      )
     end
 
     lti_integration = LtiIntegration.find_by(query_params)
-    return unauthorized_status unless lti_integration
+    if lti_integration.nil?
+      return log_unauthorized("LTI integration not found", query_params)
```

Comment:
I'm assuming this one-liner got refactored due to it being too long with the new logging function?

Reply:
I actually don't recall for sure, but yeah I probably did it because of how long the one-liner looked. Open to doing it either way

Reply:
I think I'm fine either way too. I wonder if robocop or our styleguide has a preference.